docs/04_web/03_design-systems/03_preview.md

```diff
@@ -9,7 +9,72 @@ tags:
   - preview
 ---
 
+## Introduction
+
+The Design System Configurator you see below, will be the tool we use to configure the design tokens and determine the look that's best suited for our application.
+
+On the left hand side, is the [Editor](../../../web/design-systems/preview/#editor), where we can make all the modifications we want. On the right side, we can see the [Preview](../../../web/design-systems/preview/#preview) with the results of those changes.
+
+Try changing the configurations and observe the reflected changes on the right.
+
 <design-system-editor>
   <design-system-preview></design-system-preview>
   <design-system-export slot="export"></design-system-export>
-</design-system-editor>
\ No newline at end of file
+</design-system-editor>
+
+## Editor
+
+The checkbox in the editor is ticked by default.
```

Comment:
In the above line instead of just The checkbox , may be provide a little bit of context like the "preserve selected options checkbox"